src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -41,7 +42,7 @@ def get(self, request: Request, project) -> Response:
         return self.paginate(
             request,
             paginator_cls=CombinedQuerysetPaginator,
-            on_results=lambda x: serialize(x, request.user, CombinedRuleSerializer()),
+            on_results=lambda x: serialize(x, request.user, CombinedRuleSerializer(expand=expand)),
```

Comment:
[Here](https://github.com/getsentry/sentry/blob/master/static/app/views/alerts/list/rules/alertRulesList.tsx#L49-L58) is where we could pass `expand`, I'm not sure why this wasn't passed before. I added a test for it.